spec/ddtrace/contrib/action_cable/instrumentation_spec.rb

```diff
@@ -24,7 +24,7 @@
   before do
     Datadog.configure do |c|
       c.use :rails, options
-      c.use :action_cable
+      c.use :action_cable, options
```

Comment:
This test started to fail because it was not configuring a test tracer, using the global tracer instead.
The global tracer can have traces unrelated to this test, so the total number of spans in the tracer writer might differ from expected.

In this test there was one span unrelated to the test that we were manually filtering out before, which we don't have to now anymore.